ImperatorToCK3/Copy_Files.bat

```diff
@@ -4,8 +4,9 @@ rem copy "Data_Files\ReadMe.txt" "..\Release\ImperatorToCK3\readme.txt"
 rem copy "Data_Files\ReadMe.txt" "..\Release\readme.txt"
 rem copy "Data_Files\FAQ.txt" "..\Release\ImperatorToCK3\FAQ.txt"
 
-rem Copy ImagaMagick DLLs
+rem Copy DLLs
 xcopy "..\ImageMagick\dll" "..\Release\ImperatorToCK3" /Y /E /I
+copy "Resources\rakaly.dll" "..\Release\ImperatorToCK3\"
```

Comment:
As an alternate option, you can add the dll to your VS project and set it to copy. As a plus, VS will be smart enough to only copy it when the dl is updated or on clean builds. Not a big deal for one file, but it adds up when you have a lot.

Reply:
✔️